devito/__init__.py

```diff
@@ -45,10 +48,22 @@ def _cast_and_update_compiler(val):
 configuration.add('platform', 'intel64', PLATFORMs)
 
 
+# In develop-mode:
+# - Some optimizations may not be applied to the generated code.
+# - The compiler performs more type and value checking
+def _switch_cpu(develop_mode):
+    if bool(develop_mode) is False:
+        isa, platform = infer_cpu()
+        configuration['isa'] = os.environ.get('DEVITO_ISA', isa)
+        configuration['platform'] = os.environ.get('DEVITO_PLATFORM', platform)
+    else:
+        configuration['isa'] = 'cpp'
+        configuration['platform'] = 'intel64'
```

Comment:
So develop mode only accept intel plateform?

Reply:
Just to clarify: Devito works on whatever architecture gcc/icc work, to be precise.

These are just optimization parameters. And yeah, since we mostly use intel, I just used the same names that YASK supports